build/NuGetAdditionalFiles/RunCsc

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
This file should be temporary. Also, does it matter what it's called? It won't be on the path.